seed/challenges/01-front-end-development-certification/html5-and-css.json

```diff
@@ -1691,91 +1695,6 @@
       "challengeType": 0
     },
     {
-      "id": "bad87fee1348bd9aedf08818",
```

Comment:
This challenge introduced the `alt` tag, quite a few challenges after the `img` element was introduced. 

I introduced the `alt` attribute at the same time as the `img` element (line 993), seeing as it is in the standard to have an `alt` attribute with `img`'s